packages/venia-ui/lib/components/Gallery/item.js

```diff
@@ -66,6 +68,15 @@ const GalleryItem = props => {
         </div>
     );
 
+    const ratingAverage = rating_summary ? (
```

Comment:
It would be best to refactor the ratings out into its own component (under Gallery would be fine). This will allow for easier extension and maintenance. You can move the styles out into its own file then too